users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_configurable.py

```diff
@@ -71,6 +77,9 @@ def add_specaug_layer(
         "time_mask_max_proportion": 1.0,
         "freq_mask_max_proportion": 1.0,
         "max_time_num_seq_len_divisor": 0.7,
+        "mask_based_on_variance": False,
```

Comment:
It could help for making this more generic to have something like `filter_based_masking_strategy` instead. By default it would be `None`, then nothing is done in this direction. If it is `"variance"`, then `transform_with_variance()` is used. Similarly, other strategies could be added.